sdk/src/settings.rs

```diff
@@ -437,6 +645,49 @@ pub(crate) fn get_settings_value<'de, T: serde::de::Deserialize<'de>>(
     })
 }
 
+#[allow(unused)]
+pub(crate) fn get_profile_settings_value<'de, T: serde::de::Deserialize<'de>>(
```

Comment:
Just call this get_setting_value(). The profile is a global setting and will always apply.